app/services/ssg_config_downloader.rb

```diff
@@ -3,19 +3,20 @@
 # Service for dowloading SSG configuration files
 class SsgConfigDownloader
   class << self
-    SSG_DS_FILE = File.new('config/supported_ssg.yaml')
+    FILE_PATH = 'config/supported_ssg.yaml'
```

Comment:
A bit off-topic, but can we use some temporary path? My SSD would thank you :smiley:.